concurrent_test.go

```diff
@@ -156,17 +216,13 @@ func concurrentReadAndWrite(t *testing.T,
 	}
 
 	close(stopCh)
-	t.Log("Wait for all transactions to finish.")
+	t.Log("Waiting for all transactions to finish.")
 	if err := g.Wait(); err != nil {
 		t.Errorf("Received error: %v", err)
 	}
 
-	saveDataIfFailed(t, db)
-
-	// TODO (ahrtr):
-	//   1. intentionally inject a random failpoint.
-	//   2. validate the linearizablity: each reading transaction
-	//      should read the value written by previous writing transaction.
+	sort.Sort(rs)
```

Comment:
There is no reason that runWorkers should return ordered `historyRecords`. It's `validateSerializable` that expect history to be ordered. Please move sorting there to avoid implicit dependencies on order.

Reply:
Makes sense. Done.